Core/Core/UIViews/CoreWebView.swift

```diff
@@ -138,6 +169,17 @@ open class CoreWebView: WKWebView {
         }
     }
 
+    private func addRefreshControl(color: UIColor?) {
+        scrollView.addSubview(refreshControl)
+        scrollView.bounces = true
+        refreshControl.color = color
+    }
+
+    @objc func refreshWebView(_ sender: UIRefreshControl) {
+        reload()
+        sender.endRefreshing()
```

Comment:
Would it be possible to call this in `public func webView(_ webView: WKWebView, didFinish navigation: WKNavigation!) {` so the spinner would animate until the refresh navigation is finished. If `reload()` gives back a `WKNavigation` we could track exactly when the refresh navigation finishes.

Reply:
Fixed!